packages/oui-collapsible/_mixins.less

```diff
@@ -0,0 +1,96 @@
+@import './_variables';
+
+#oui {
+  .collapsible(
```

Comment:
(optional) You can put your style directly in the main file. We stopped structuring our styles with this kind of mixin (called once) :)

Here for example: https://github.com/ovh-ux/ovh-ui-kit/blob/master/packages/oui-stepper/stepper.less

Reply:
Sure. Thanks for the comment. I will follow this from my next PR onwards :)